### ISSUE TITLE
Support change of service generation for TTS

### ISSUE THREAD

**Issue by zoic21** [duncan3dc/sonos#22]
Hi,

It's possible to change (or allow user to specify is own service) for TTS, Google is pretty limited but voxygen not : 
````
$url = Helper::url("http://www.voxygen.fr/sites/all/modules/voxygen_voices/assets/proxy/index.php", [
                "method"   =>  $this->method,
                "voice"   =>  $this->voice,
                "text"   =>  $this->text,
            ]);
````

Thank in advance

**Comment by duncan3dc** [comment]
Yes I've been working on this since release, I'll let you know when it's ready

**Comment by zoic21** [comment]
Perfect, thank

**Comment by duncan3dc** [comment]
This will make it into the next release (1.3), you can test out the development version like so:

```
composer require duncan3dc/sonos:dev-speaker-integration
```

```php
$provider = new \duncan3dc\Speaker\Providers\VoxygenProvider;
$provider->setVoice("Bronwen");

$track = new TextToSpeech("Hello", $directory);
$track->setProvider($provider);

$controller->interrupt($track, 50);
```

The text-to-speech conversion will now be handled by the [duncan3dc/speaker](https://github.com/duncan3dc/speaker) project

**Comment by duncan3dc** [comment]
This is available in [version 1.3.1](http://duncan3dc.github.io/sonos/fun/text-to-speech/) now
